tests/aws/services/cloudformation/test_template_engine.py

```diff
@@ -235,20 +235,23 @@ def test_cidr_function(self, deploy_cfn_template):
         assert deployed.outputs["Address"] == "10.0.0.0/24"
 
     @markers.aws.validated
-    @pytest.mark.skip(reason="function not currently supported")
-    def test_get_azs_function(self, deploy_cfn_template):
+    def test_get_azs_function(self, deploy_cfn_template, snapshot):
+        """
+        TODO parametrize this test.
+        For that we need to be able to parametrize the client region. The docs show the we should be
+        able to put any region in the parameters but it doesn't work. It only accepts the same region from the client config
+        if you put anything else it just returns an empty list.
```

Comment:
:+1: 

I think it's fine for now since we usually assume us-east-1 anyway, i.e. we don't currently handle region specific idiosyncrasies and treat everything as if it were us-east-1, which has the broadest support out of all regions.